roles/nova-common/defaults/main.yml

```diff
@@ -23,7 +27,7 @@ nova:
   python_libvirt_version: 1.2.2-0ubuntu2~cloud0
   qemu_kvm_version: 2.0.0+dfsg-2ubuntu1.16~cloud0
   librdb1_version: 0.80.9-0ubuntu0.14.04.2~cloud0
-  glance_endpoint: http://{{ endpoints.glance }}:9292
+  glance_endpoint: "{{ endpoints.glance.url.internal }}"
```

Comment:
This would change us from connecting to glance via http directly to the glance service to going through haproxy via https. Is this intended? 

We've had some discussion as to why nova is connecting to glance outside of haproxy, but I don't recall the reason right now.